### PR TITLE
Update information-about-the-outlook-autocomplete-list.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the article to include outlook 2019

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/OfficeDocs-Support/issues/1092

File: Outlook/Client/contacts/information-about-the-outlook-autocomplete-list.md [hunk 1]
```diff
@@ -57,7 +58,7 @@ The limits are as follows:
 
 This section details how to enable or disable the AutoComplete feature.
 
-#### Outlook 2010, Outlook 2013, and Outlook 2016
+#### Outlook 2010, Outlook 2013, Outlook 2016 and Outlook 2019
```
Comment (JohanFreelancer9) [inline Outlook/Client/contacts/information-about-the-outlook-autocomplete-list.md:61]:
```suggestion
#### Outlook 2010, Outlook 2013, Outlook 2016, and Outlook 2019
```

File: Outlook/Client/contacts/information-about-the-outlook-autocomplete-list.md [hunk 2]
```diff
@@ -78,19 +79,19 @@ To access the AutoComplete settings, follow the steps:
 
     :::image type="content" source="media/information-about-the-outlook-autocomplete-list/enable-the-autocomplete-feature-for-outlook-2003-and-2007.jpg" alt-text="Advanced E-mail Options window, Suggest names while completing To, Cc, and Bcc fields checked" border="false":::
 
-### How to import .nk2 files into Outlook 2010, Outlook 2013, and Outlook 2016
+### How to import .nk2 files into Outlook 2010, Outlook 2013, Outlook 2016 and Outlook 2019
```
Comment (JohanFreelancer9) [inline Outlook/Client/contacts/information-about-the-outlook-autocomplete-list.md:82]:
```suggestion
### How to import .nk2 files into Outlook 2010, Outlook 2013, Outlook 2016, and Outlook 2019
```

File: Outlook/Client/contacts/information-about-the-outlook-autocomplete-list.md [hunk 3]
```diff
@@ -78,19 +79,19 @@ To access the AutoComplete settings, follow the steps:
...
 
-Microsoft Office Outlook 2007 and earlier versions store the AutoComplete list in a nickname (.nk2) file on the disk. Outlook 2010, Outlook 2013, and Outlook 2016 store the AutoComplete list as a hidden message in your primary message store. Outlook 2010, Outlook 2013, and Outlook 2016 let you import the older .nk2 files.
+Microsoft Office Outlook 2007 and earlier versions store the AutoComplete list in a nickname (.nk2) file on the disk. Outlook 2010, Outlook 2013, and Outlook 2016 store the AutoComplete list as a hidden message in your primary message store. Outlook 2010, Outlook 2013, Outlook 2016  and outlook 2019 let you import the older .nk2 files.
```
Comment (JohanFreelancer9) [inline Outlook/Client/contacts/information-about-the-outlook-autocomplete-list.md:84]:
```suggestion
Microsoft Office Outlook 2007 and earlier versions store the AutoComplete list in a nickname (.nk2) file on the disk. Outlook 2010, Outlook 2013, and Outlook 2016 store the AutoComplete list as a hidden message in your primary message store. Outlook 2010, Outlook 2013, Outlook 2016, and outlook 2019 let you import the older .nk2 files.
```

File: Outlook/Client/contacts/information-about-the-outlook-autocomplete-list.md [hunk 4]
```diff
@@ -78,19 +79,19 @@ To access the AutoComplete settings, follow the steps:
...
 
 For more information about how to import .nk2 files in Outlook 2010, see [Import Auto-Complete List from another computer](https://support.microsoft.com/office/import-or-copy-the-auto-complete-list-to-another-computer-83558574-20dc-4c94-a531-25a42ec8e8f0).
 
 #### How to copy the AutoComplete list
 
 The steps to export and import the AutoComplete list are different, depending on the version of Outlook that you're using.
 
-##### Outlook 2016, Outlook 2013, and Outlook 2010
+##### Outlook 2010, Outlook 2013, Outlook 2016 and Outlook 2019
```
Comment (JohanFreelancer9) [inline Outlook/Client/contacts/information-about-the-outlook-autocomplete-list.md:92]:
```suggestion
##### Outlook 2010, Outlook 2013, Outlook 2016, and Outlook 2019
```

File: Outlook/Client/contacts/information-about-the-outlook-autocomplete-list.md [hunk 5]
```diff
@@ -78,19 +79,19 @@ To access the AutoComplete settings, follow the steps:
...
 
-To copy the AutoComplete list in Outlook 2016, Outlook 2013, and Outlook 2010, follow these steps:
+To copy the AutoComplete list in Outlook 2010, Outlook 2013, Outlook 2016 and Outlook 2019 follow these steps:
```
Comment (JohanFreelancer9) [inline Outlook/Client/contacts/information-about-the-outlook-autocomplete-list.md:94]:
```suggestion
To copy the AutoComplete list in Outlook 2010, Outlook 2013, Outlook 2016, and Outlook 2019, follow these steps:
```

File: Outlook/Client/contacts/information-about-the-outlook-autocomplete-list.md [hunk 6]
```diff
@@ -161,7 +162,7 @@ Selecting **X** will prevent that entry (name) from appearing in the AutoComplet
 
 This section explains how to delete the AutoComplete list.
 
-##### Outlook 2016, Outlook 2013, and Outlook 2010
+##### Outlook 2010, Outlook 2013, Outlook 2016 and Outlook 2019
```
Comment (JohanFreelancer9) [inline Outlook/Client/contacts/information-about-the-outlook-autocomplete-list.md:165]:
```suggestion
##### Outlook 2010, Outlook 2013, Outlook 2016, and Outlook 2019
```

File: Outlook/Client/contacts/information-about-the-outlook-autocomplete-list.md [hunk 7]
```diff
@@ -161,7 +162,7 @@ Selecting **X** will prevent that entry (name) from appearing in the AutoComplet
...
 
 Use one of the following two methods to delete the AutoComplete list in Outlook 2010, Outlook 2013, and Outlook 2016.
```
Comment (JohanFreelancer9) [inline Outlook/Client/contacts/information-about-the-outlook-autocomplete-list.md:167]:
```suggestion
Use one of the following two methods to delete the AutoComplete list in Outlook 2010, Outlook 2013, Outlook 2016, and Outlook 2019.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, please commit my suggested changes. Many thanks.

**Pull request merged**